db/lib/database-access.js

```diff
@@ -0,0 +1,101 @@
+/** @import { BlockEvent } from '../../backend/lib/rpc-service/data-types.js' */
+/** @import { Static } from '@sinclair/typebox' */
+/** @import {Queryable} from '@filecoin-station/deal-observer-db' */
+import { ActiveDealDbEntry } from '@filecoin-station/deal-observer-db/lib/types.js'
+import { Value } from '@sinclair/typebox/value'
+
+/**
+ * @param {Queryable} pgPool
+ * @returns {Promise<ActiveDealDbEntry | null>}
```

Comment:
Please apply the changes from https://github.com/filecoin-station/deal-observer/commit/b6cf42ec3802b4fb8171b08530581a510a783a33 to fix the typings.